src/ai/backend/common/web/session/__init__.py

```diff
@@ -98,6 +101,21 @@ def created(self) -> int:
     def empty(self) -> bool:
         return not bool(self._mapping)
 
+    @property
+    def expires(self) -> Optional[str]:
+        return self._expires
+
+    @expires.setter
+    def expires(self, value: Optional[str]) -> None:
+        if value is None and self.max_age is not None:
+            expires_from_now = datetime.fromtimestamp(
+                time.time() + int(self.max_age), tz=timezone.utc
+            )
+            self._expires = expires_from_now.strftime("%a, %d-%b-%Y %H:%M:%S GMT")
```

Comment:
Could we use locale-aware datetime representation?
Or, could we use some standard format like ISO8601 and let the frontend convert it to a locale-aware representation?